Cargo.toml

```
@@ -1,7 +1,17 @@
 [package]
 name = "ndarray-stats"
 version = "0.1.0"
-authors = ["Jim Turner <rust@turner.link>"]
+authors = ["Jim Turner <rust@turner.link>", "LukeMathWalker <lpalmieri@contact.com>"]
```

Will you please change my email address at the same time?
```suggestion
authors = ["Jim Turner <ndarray-stats@turner.link>", "LukeMathWalker <lpalmieri@contact.com>"]
```